telemetry.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, request
+from influxdb import InfluxDBClient
+
+app = Flask(__name__)
+
+client = InfluxDBClient(host='localhost', port=8086)
```

Comment:
I think in a dockerized env this will need to used service name of the influx container,
maybe we need some config to be extracted

Reply:
I see @tot-ra I will look into it